code/datums/mutations/holy_mutation/honorbound.dm

```diff
@@ -0,0 +1,251 @@
+		if(job in GLOB.security_positions || is_holy)
+			to_chat(honorbound_human, "<span class='warning'>There is nothing righteous in attacking the <b>just</b>.</span>")
+			return FALSE
+		if(job in GLOB.medical_positions)
+			to_chat(honorbound_human, "<span class='warning'>If you truly think this healer is not <b>innocent</b>, declare them guilty.</span>")
+			return FALSE
+	//THE INNOCENT
+	if(!is_guilty)
+		to_chat(honorbound_human, "<span class='warning'>There is nothing righteous in attacking the <b>innocent</b>.</span>")
+		return FALSE
+	return TRUE
+
+// SIGNALS THAT ARE FOR BEING ATTACKED FIRST (GUILTY)
+/datum/mutation/human/honorbound/proc/attackby_guilt(datum/source, obj/item/I, mob/attacker)
+	SIGNAL_HANDLER
+	if(I.force && I.damtype != STAMINA)
+		guilty(attacker)
+
+/datum/mutation/human/honorbound/proc/hulk_guilt(datum/source, mob/attacker)
+	SIGNAL_HANDLER
+	guilty(attacker)
+
+/datum/mutation/human/honorbound/proc/hand_guilt(datum/source, mob/living/attacker)
+	SIGNAL_HANDLER
+	if(attacker.combat_mode)
+		guilty(attacker)
+
+/datum/mutation/human/honorbound/proc/paw_guilt(datum/source, mob/living/attacker)
+	SIGNAL_HANDLER
+	guilty(attacker)
+
+/datum/mutation/human/honorbound/proc/bullet_guilt(datum/source, obj/projectile/Proj)
+	SIGNAL_HANDLER
+	var/mob/living/shot_honorbound = source
+	if(istype(Proj, /obj/projectile/beam) || istype(Proj, /obj/projectile/bullet) || istype(Proj, /obj/projectile/magic))
+		if((Proj.damage_type != STAMINA))
+			if(!Proj.nodamage && Proj.damage < shot_honorbound.health && isliving(Proj.firer))
+				guilty(Proj.firer)
+
+/datum/mutation/human/honorbound/proc/thrown_guilt(datum/source, atom/movable/thrown_movable, skipcatch = FALSE, hitpush = TRUE, blocked = FALSE, datum/thrownthing/throwingdatum)
+	SIGNAL_HANDLER
+	if(istype(thrown_movable, /obj/item))
+		var/mob/living/honorbound = source
+		var/obj/item/thrown_item = thrown_movable
+		if(thrown_item.throwforce < honorbound.health && ishuman(thrown_item.thrownby))
+			guilty(thrown_item.thrownby)
+
+//spell checking
+/datum/mutation/human/honorbound/proc/spell_check(mob/user, obj/effect/proc_holder/spell/spell_cast)
+	SIGNAL_HANDLER
+	punishment(user, spell_cast.school)
+
+/datum/mutation/human/honorbound/proc/staff_check(mob/user, obj/item/gun/gun_fired, target, params, zone_override)
+	SIGNAL_HANDLER
+	if(!istype(gun_fired, /obj/item/gun/magic))
+		return
+	var/obj/item/gun/magic/offending_staff = gun_fired
+	punishment(user, offending_staff.school)
+
+/datum/mutation/human/honorbound/proc/punishment(mob/living/carbon/human/user, school)
+	switch(school)
+		if(SCHOOL_EVOCATION, SCHOOL_TRANSMUTATION, SCHOOL_CONJURATION, SCHOOL_TRANSMUTATION)
+			to_chat(user, "<span class='userdanger'>[GLOB.deity] is angered by your use of [school] magic!</span>")
+			lightningbolt(user)
+			SEND_SIGNAL(owner, COMSIG_ADD_MOOD_EVENT, "honorbound", /datum/mood_event/holy_smite)//permanently lose your moodlet after this
+		if(SCHOOL_NECROMANCY, SCHOOL_FORBIDDEN)
```

Comment:
should put this ifrst and then just else the rest, or otherwise this is a list that has to be maintained if people add schools of magic.